src/config/config.c

```diff
@@ -983,6 +1025,19 @@ cfgOptionSet(ConfigOption optionId, ConfigSource source, const Variant *value)
         // Only set value if it is not null
         if (value != NULL)
         {
+            // If this option is in a group then set the value in the group for the option's index. This indicates that there is at
+            // least one option set in the group for that index.
+
```

Comment:
Remove extra blank line between comment and associated IF statement.

Reply:
Removed.